src/scaffoldmaker/meshtypes/meshtype_3d_trigeminalnerve1.py

```diff
@@ -0,0 +1,591 @@
+"""
+Generates a 3D trigeminal nerve scaffold using tube network mesh.
+"""
+from cmlibs.maths.vectorops import cross, mult, set_magnitude, sub
+from cmlibs.utils.zinc.field import Field, find_or_create_field_coordinates
+from cmlibs.zinc.node import Node
+from scaffoldmaker.annotation.annotationgroup import AnnotationGroup
+from scaffoldmaker.annotation.trigeminal_nerve_terms import get_trigeminal_nerve_term
+from scaffoldmaker.meshtypes.meshtype_1d_network_layout1 import MeshType_1d_network_layout1
+from scaffoldmaker.meshtypes.scaffold_base import Scaffold_base
+from scaffoldmaker.scaffoldpackage import ScaffoldPackage
+from scaffoldmaker.utils.interpolation import (getCubicHermiteArcLength, interpolateLagrangeHermiteDerivative,
+                                               sampleCubicHermiteCurvesSmooth)
+from scaffoldmaker.utils.networkmesh import NetworkMesh
+from scaffoldmaker.utils.tubenetworkmesh import BodyTubeNetworkMeshBuilder, TubeNetworkMeshGenerateData
```

Comment:
Groups dorsal/ventral/left/right don't make any sense. I think you only need to derive from TubeNetworkMeshBuilder to remove them. [I don't want to do this at the moment, but technically dorsal/ventral could become left/right or medial lateral; the latter are painful as you'd need to define it it's left or right trigeminal.]